src/index.js

```diff
@@ -121,8 +121,11 @@ class Libp2p extends EventEmitter {
       })
     }
 
-    // enable/disable pubsub
-    if (this._config.EXPERIMENTAL.pubsub) {
+    // start pubsub
+    if (this._config.pubsub.enabled) {
+      const Pubsub = this._modules.pubsub
+
+      this._pubsub = new Pubsub(this)
       this.pubsub = pubsub(this)
```

Comment:
What do you think about just doing `this.pubsub = pubsub(this, this._modules.pubsub)` and letting `./src/pubsub.js` take care of keeping track of pubsub? I think it gets confusing to have both _pubsub and pubsub. I was having a hard time keeping track of what was what.

Reply:
Seems better your suggestion. Will refactor it now